common/src/crs.rs

```diff
@@ -5,14 +5,17 @@ const G1_START: usize = 28;
 const G2_START: usize = 28 + (5_040_001 * 64);
 const G2_END: usize = G2_START + 128 - 1;
 
+const BACKEND_IDENTIFIER: &str = "aztec_barretenberg_ultraplonk";
```

Comment:
```suggestion
const BACKEND_IDENTIFIER: &str = "barretenberg_ultraplonk";
```
Are you good with removing aztec or do you think there will be "other barretenbergs"?